nodeJS/mongoDB/intro_to_mongoDB.md

```diff
@@ -1,6 +1,12 @@
 ### Introduction
 
-Data persistence is integral to the vast majority of web applications. Local storage has limitations and poses security risks, and SaaS options such as Firebase work well, but ultimately being able to understand how to structure, build, and query your own database are important skills for any full stack developer to have. For example, consider X (formerly known as Twitter). Where are your username and password stored? Or perhaps where all of your posts might be fetched from? How does the application determine what posts are shown to you every time you log on? These operations are handled by a database. Databases are split into "relational" and "non-relational" types of databases, and each handles data and scaling in different manners. [This article](https://circleci.com/blog/SQL-vs-NoSQL-databases/) goes in depth with the difference between the two. This lesson will teach you about MongoDB, a popular NoSQL database.
+Data persistence is integral to the vast majority of web applications. Local storage has limitations and poses security risks, and Backend-as-a-Service (BaaS) options such as Firebase work well, ultimately being able to understand how to structure, build, and query your own database are important skills for any full-stack developer to have.
+
+For example, consider X (formerly known as Twitter). Where are your username and password stored? Or perhaps where all of your posts might be fetched from? How does the application determine what posts are shown to you every time you log on? These operations are handled by a database.
+
+Databases are split into "relational" and "non-relational" types of databases, and each handles data and scaling in different manners.
+
+In this lesson, you will complete a course that teaches you about MongoDB, a popular non-relational database.
```

Comment:
```suggestion
Databases are split into two types - "relational" and "non-relational", and each type handles data and scaling in different manners. In this lesson, you will complete a course that teaches you about MongoDB, a popular non-relational database.
```
I like moving the article to the assignment section - great change.
With that change, combining these two separate parts makes sense again. A little wording suggestion as well just to improve the sentence flow.

Reply:
Nice!